app/models/setting/puppet.rb

```diff
@@ -1,35 +1,7 @@
+# Do not use - these settings are now in Setting::Configuration, but this model
+# has to stay in order to be able to migrate them
```

Comment:
This won't work, unfortunately.  Rails doesn't load, because early in initialization it sees the STI column references a class that doesn't exist and then bails out.  That happens long before the migration gets loaded.